administrator/language/de-DE/com_config.ini

```
@@ -134,8 +134,9 @@ COM_CONFIG_FIELD_PROXY_PORT_LABEL="Port"
 COM_CONFIG_FIELD_PROXY_USERNAME_LABEL="Benutzer"
 COM_CONFIG_FIELD_REDIS_AUTH_LABEL="Redis-Server-Authentifizierung"
 COM_CONFIG_FIELD_REDIS_DB_LABEL="Redis-Datenbank"
-COM_CONFIG_FIELD_REDIS_HOST_LABEL="Redis-Server"
+COM_CONFIG_FIELD_REDIS_HOST_LABEL="Redis-Server Host/Sock"
```

```suggestion
COM_CONFIG_FIELD_REDIS_HOST_LABEL="Redis-Server Host/Socket"
```

einen "Redis Sock" scheint es nicht zu geben?

---

was haben die sich im Core dann wieder gedacht? Nichts evtl.